nubis/puppet/consul.pp

```diff
@@ -69,6 +69,13 @@
       verbose => true,
       redownload => true, # The file already exists, we replace it
```

Comment:
So the `$mode` parameter is available but only if you use the `$cache_dir` option additionally the latest release of this module is `1.7` which does not include the changes to include the `mode` parameter.

Our options are:
1. Pin Puppetfile to a git hash instead of Version 1.7 (this is not the best solution)
2. Use this PR for now until they get a release out

The path of least resistance for now is to just add file resource like it did after `wget::fetch` to enforce file permissions